core/src/main/java/org/frankframework/management/bus/endpoints/ServerStatistics.java

```diff
@@ -211,6 +211,12 @@ public Message<String> getApplicationWarnings() {
 		return new JsonMessage(returnMap);
 	}
 
+	@ActionSelector(BusAction.UPDATES)
+	@PermitAll
+	public Message<String> getApplicationWarningsUpdates(){
+		return getApplicationWarnings();
+	}
```

Comment:
Wat is hier het nut van?

Reply:
The current authorisation works by checking the user's roles as if it was a http request with user session, but this information doesn't exist when the backend triggers a scheduled method and sending a gateway message to retrieve the data from core so I had to make this or find a way to put the right roles on a non-user to still get past role authorisation